Add radio buttons and corresponding Field/FieldGroup enhancements

kschiffer opened the pull request:
**Summary:**
Closes #353, as well as some additional changes necessary for #332 (see below).

**Changes:**
- Add `RadioButton` component
- Setup `Field` component to support radio buttons
- Setup `FieldGroup` component to support radio buttons
- Fixes for proper `horizontal` display of `FieldGroup`
- `column` option for `FieldGroup`
- Styling fixes
- Add a change handler prop to `Field` to hook into field changes (we'll need that for #332)
- Add corresponding stories to `Field` and `Form`

**Notes for Reviewers:**
The `FieldGroup` will handle the radio-buttons values and map it's `name` prop

bafonins commented on pkg/webui/components/field/group/index.js:
```diff
@@ -13,39 +13,58 @@
+            appliedProps.checked = value === fieldValue
+          }
+        }
+        const classNames = classnames(style.field, fieldClassName, groupClassName, {
```
Do we want to apply `groupClassName` to the `<Field />` component?

bafonins commented on pkg/webui/components/radio-button/index.js:
```diff
@@ -0,0 +1,59 @@
+    )
+  }
+}
+
```
redundant new line

kschiffer commented:
> 1. radio buttons dont focus when one is checked

I was confused here as well, but this is actually W3C behavior. One can tab into the radio button group 
and then select using the arrow keys. I have adjusted the styling a little to make the focus state clearer.
> 2. <FieldGroup /> validation is broken. After debugging it seems that the problem is related to the touched object which is undefined for the group.

Fixed. Validation was working, it was just that the error was not displayed. This was indeed related to the touched prop not being set properly.
